### PR TITLE
Fixing missing services

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves N/A

**Overall change:** 

This PR updates fixture data for Thai as it was was missing the promo objects in the fixture data causing the front page to not render.

Also refactors the service to have a single function that generates the data file paths to to enable adding in a service override for the cymrufyw service.

**Code changes:**

- Updates fixture data
- Updates snapshots based on new fixture data
- Add `constructDataFilePath` to generate the data file paths including any potential service overrides.
- Add a service override key value JS object

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by hindsc52** [comment]
image

**Comment by chris-hinds** [comment]
All services now returning a 200 response in simorgh for front pages

image

File: src/server/serviceOverrides.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const serviceOverrides = {
+  cymrufyw: 'newyddion',
```
Comment (dr3) [inline src/server/serviceOverrides.js]: Does this mean simorgh internally serves the `data/naidheachdan/frontpage/index.json` under `/cymrufyw.json`? Do we want that? I thought we would request `/newyddion.json` as thats how the url will be IRL

Comment (dr3) [reply src/server/serviceOverrides.js]: I guess it depends where we do the `newyddion` -> `cymrufyw` mapping, if we do it in mozart, we should just have the local data file be `data/cymrufyw/frontpage/index.json`, if we do it in simorgh, we should serve `/newyddion.json` and do the logic switching inside `getInitalData`

Comment (chris-hinds) [reply src/server/serviceOverrides.js]: ahhh of course, this will only work on localhost when serving the fixture data

Comment (chris-hinds) [reply src/server/serviceOverrides.js]: doing it inside of mozart would be nice, but I don't think we would be able to do that anytime soon.

Would be easier to do inside of simorgh in the getInitialData method for now

Comment (chris-hinds) [reply src/server/serviceOverrides.js]: I'll change this up tomorrow and do the mapping in getInitialData

Comment (chris-hinds) [reply src/server/serviceOverrides.js]: This override logic has now been removed, as we will need to implement this in Mozart routing.

**Review by dr3 — APPROVED** [review]

**Review by thekp — APPROVED** [review]
🥇

**Comment by PriyaKR** [comment]
LGTM.

**Pull request merged**